src/Symfony/Component/Notifier/Bridge/Smsapi/README.md

```diff
@@ -7,12 +7,13 @@ DSN example
 -----------
 
 ```
-SMSAPI_DSN=smsapi://TOKEN@default?from=FROM
+SMSAPI_DSN=smsapi://TOKEN@default?from=FROM&fast=FAST
 ```
 
 where:
  - `TOKEN` is your API Token (OAuth)
  - `FROM` is the sender name
+ - `FAST` setting this parameter to "1" (default "0") will result in sending message with the highest priority which ensures the quickest possible time of delivery. Attention! Fast messages costs 50% more than normal message.
```

Comment:
I suggest saying `Fast messages cost more than normal messages`, without mentioning the actual increase of 50%. this way, this readme does not become outdated if they change their pricing to modify the ratio.

Reply:
@stof You are right. Done.